apps/settings/js/icc_menu.js

```diff
@@ -72,11 +72,21 @@
 
         var li = document.createElement('li');
         var a = document.createElement('a');
+        var commandIcons = menu[SIMNumber].entries.icons;
+        var icon = commandIcons && commandIcons.length > 0 ?
+          commandIcons[0] : null;
 
         a.id = 'menuItem-icc-' + menu[SIMNumber].iccId;
         a.className = 'menu-item menuItem-icc';
         a.href = '#icc';
-        a.dataset.icon = 'sim-toolkit';
+        if (icon && STKHelper) {
```

Comment:
With requirejs, we don't have to check `STKHelper` anymore because it would always exist.